typescript/introduction/intro-typescript/what-is-typescript.md

```diff
@@ -0,0 +1,44 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Intro to TypeScript
+
+---
+## Content
+
+If you're old enough, you might remember when JavaScript was just a web development language. Due to its versatility, JavaScript is now used in many applications including neural network design, power backends and many more.
+
+You're probably thinking "What's JavaScript got to do with it -- isn't this a TypeScript course?". Let us explain!
+
+Because of the high adoption rate of JavaScript, many tools (such as Angular, React, Node.js, etc) were created to allow programmers to write better/safer JavaScript code. 
+
+TypeScript is another creation meant to put the future of JavaScript in the hands of developers, and it's actually a **superset of JavaScript**. What that means is that TypeScript has all the functionality of JavaScript, but *adds even more*.
+
```

Comment:
We can add an image here?

[typescript-is-superset-of-javascript](https://img.enkipro.com/074c0352be386e58bbdd697ed8bff427.png)

```suggestion

[typescript-is-superset-of-javascript](https://img.enkipro.com/074c0352be386e58bbdd697ed8bff427.png)
```